.ci_scripts/beforeInstall.sh

```diff
@@ -91,7 +91,7 @@ mkdir /tmp/mongodb
 SSL_PASS=""
 
 if [ "$MONGO_PROFILE" = "self-ssl" -o "$MONGO_PROFILE" = "mutual-ssl" ]; then
-    SSL_PASS=`uuidgen`
+    SSL_PASS=secret
```

Comment:
If we really do want to generate something on the fly I've tested the following on the traivs docker container and it works
`date | sha1sum | cut -f 1 -d ' '`